src/converters/form-converter.js

```diff
@@ -0,0 +1,18 @@
+import Form from '../model/form';
+
+const formConverter = thing => {
+	const id = thing.id;
+	const title = thing.attributes.title;
```

Comment:
You can use destructuring here :P

Reply:
`const {attributes: { title }} = thing` is this way? Seems a bit confuse to read xD

Reply:
but you can do the 2 assignments in the same line

Reply:
`const {id, {attributes: {title}} = thing`?

Reply:
I was thinking on the next line, const description = thing.attributes.description;

And doing something like: const {title, description} = thing.attributes